### DISCUSSION TITLE
how can I open the search button

### DISCUSSION THREAD

**Discussion by thisguyQUACKS** [valinet/ExplorerPatcher#2244] [Q&A]
I cant open the search button. It was working now it doesn't help me pls

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by daville50** [reply]
more specifically, and me too, start menu will not search using "just type" feature and cannot click into the search box on start menu set on win 10 mode, will amend later with win ver and patcher ver, issue has persisted for several days now. All win-11 and patcher on current versions fully patched 10-17-2023. I have reviewed my patcher setting and win-11 pro setting and I cannot find anything I did to terminate search functionality. I do have patcher set for taskbar on left side but that is about all. Reboot done, restart patcher done, have not done uninstall and reinstall yet.  All I get when I type on keyboard into a start menu is the first letter recognition and then a jump to any app on the start for that letter, what should happen is my typing should show in the search box and it does not. hope this helps

**Reply by pyrates999** [reply]
Please provide the information asked for.  Not "I have the latest" because that doesn't provide any details.

And show screen shots of any settings you changed in EP.

**Comment by daville50** [comment]
Sorry I was in a hurry this am. Explorer Patcher 22621.2361.58.4, Win 11 Pro 22h2 OS Build 2261.2428. The problem persists, has for about a week or so. I absolutely love this product and if this my error somewhere in my setting I am so sorry.

Dave Manville

________________________________

**Reply by pyrates999** [reply]
Please show screen shots of any settings you changed in EP.

**Comment by daville50** [comment]
Here are screenshots of ALL settings as of Oct 18 2023
[cid:f469f4c4-2f43-43c3-bba8-2018b66fe102]
[cid:bb599616-cdac-4354-ba86-b92c0fd2cf58]
[cid:3a986686-d6b3-4c8d-815f-cec103264c17]
[cid:e45a750c-0533-4f07-b1e6-d543c15980e1]
[cid:4cac9562-f1fb-45a3-a286-8968d738e323]
[cid:44516b87-2be1-4a6f-9354-7cac50fb545d]
[cid:a0d3f3ab-f1f9-46c6-8371-ff4513f81fee]
[cid:2dc6bccf-df0a-48bc-9028-63d2a2f19c70]
[cid:5f670fb0-ca70-4e6d-9045-e9b9ebda5cba]
[cid:3fbbdf1d-40e3-4b70-99e9-7309b7c3b045]
[cid:5b7c4080-8a2e-4548-b59e-5e1c40bab2fe]

More info....
I am on a 2 Monitor System, 2 27in 1080p displays, Asus

Where Explorer Patcher lives and files and dates since last update Oct 7 2023
[cid:f65d1765-3d09-44c3-afd5-f9d657b5dd56]
[cid:6d14dad7-b7e2-4930-a77f-3ce337f0c4dd]

The search issue persists as of today, hope this helps, let me know if want more info, registry settings whatever and thanks so much for looking into this, GREAT SOFTWARE with a little bug.

Dave Manville
________________________________

**Reply by pyrates999** [reply]
You can't upload screen shots by replying via email.

**Comment by daville50** [comment]
Here are the screenshots as a Zip folder. I am not very familiar with Github. I can send a link to my folder if this Zip does not work or tell me how to get you the screen shots.

Let me know

Dave Manville
________________________________

**Reply by pyrates999** [reply]
Attachments do not show up when replied to via email.  You must log into github.

**Comment by Oooch** [comment]
I'm also on 22621.2428 and have now stopped being able to search for stuff in my Start Menu

Only options I've really changed in Explorer Patcher are to make it more centered

**Reply by pyrates999** [reply]
try this to rebuild the search index: https://learn.microsoft.com/en-us/troubleshoot/windows-client/shell-experience/windows-search-performance-issues

**Reply by Oooch** [reply]
Doesn't seem to help, you physically can't type into the search box any more, not sure if its a windows update or not

**Reply by pyrates999** [reply]
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by Oooch** [reply]
I'm on version 22621.2428.59.1 which I just updated to and I used the big reset all settings to default button and the issue persists

Very odd

**Reply by pyrates999** [reply]
did you restart file explorer?

**Reply by Oooch** [reply]
I just uninstalled ExplorerPatcher and the issues persists, seems Windows have released an update that broke something and its unrelated to this software

Thanks for your time!

**Reply by pyrates999** [reply]
If you want to repair your windows install, please run the following in an elevated command prompt or elevated powershell:

```
chkdsk C:
```

If it tells you there are errors, then run this:

```
chkdsk C: /F
```

Reboot to run it on boot.  Keep running that command until it reports no errors.  Then run this to repair windows:

```
sfc /scannow
dism /online /cleanup-image /scanhealth
dism /online /cleanup-image /restorehealth
dism /online /cleanup-image /checkhealth
dism /online /cleanup-image /analyzecomponentstore
dism /online /cleanup-image /startcomponentcleanup
```

Then reboot your computer.

**Comment by daville50** [comment]
#2244 is no longer an issue, search now works. Nov 1 2023, I upgraded to Explorer Patcher 22621.2428.59.1 and Windows 11 Pro
Version 22H2 Installed on ‎9/‎18/‎2023 OS build 22621.2506 Experience Windows Feature Experience Pack 1000.22677.1000.0 2023-10 Cumulative Update Preview for Windows 11 Version 22H2 for x64-based Systems (KB5031455) 11/1/2023.

So, the big issue above is now resolved, but a little aesthetic issue remains of low importance. With my task bar set to left side and I click into the start menu search box the new search dialog jumps into place ready to go, but it is a jarring operation and shows up at the bottom left of display, it would be nice if the search dialog could pick-up the start menu location at the top left of display and then things would be all smooth and lovely. Again, this is a very minor annoyance. Thanks so much for this software.

Dave Manville

________________________________

**Reply by pyrates999** [reply]
Please open a new discussion.
